aptiche/lib/widgets/HomeScreen/contact_card.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:aptiche/services/third_party_services.dart';
+import 'package:aptiche/utils/theme.dart';
+import 'package:aptiche/utils/ui_scaling.dart';
+import 'package:flutter/material.dart';
+
+class ContactCard extends StatelessWidget {
+  const ContactCard(
+      {Key? key,
+      required this.name,
+      required this.contact,
+      required this.context,
+      required this.thirdPartyServices})
+      : super(key: key);
+  final String name;
+  final String contact;
+  final BuildContext context;
+  final ThirdPartyServices thirdPartyServices;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(8.0),
+      child: InkWell(
+        onTap: () {
+          if (contact != "") {
+            thirdPartyServices.urlLauncher(contact);
```

Comment:
URL Launcher isn't working

Reply:
it is working for me. Also separated the privacy policy into a separate file. `assets/markdown/privacy_policy.md`